libsolidity/Parser.cpp

```diff
@@ -116,6 +116,7 @@ ASTPointer<ContractDefinition> Parser::parseContractDefinition()
 {
 	ASTNodeFactory nodeFactory(*this);
 	ASTPointer<ASTString> docString;
+	bool contractFullyImplemented = true;
```

Comment:
For the contract, this should be a type information:
If a contract does not overwrite all non-implemented functions from the base, it is considered "abstract".
These contracts cannot be compiled to a "final contract" and cannot be used in "new" expressions, which should be checked at compilation time.